frb_rust/src/dart_api_dl/include/dart_api.h

```diff
@@ -1302,7 +1302,7 @@ DART_EXPORT void Dart_DisableHeapSampling();
  *   Dart_DeleteWeakPersistentHandle once the embedder no longer needs it.
  *
  * - The provided callback must not call into the VM and should do as little
- *   work as possible to avoid performance penalities.
+ *   work as possible to avoid performance penalties.
```

Comment:
dart_api_dl is 3rd party IIRC, so we cannot change it

Reply:
Oh, I rollbacked it. 😵